src/config.rs

```diff
@@ -102,6 +102,14 @@ impl Templates {
         Template(mustache::compile_path("tmpl/confirm_email.mustache").unwrap())
     }
 
+    fn default_email_html() -> Template {
```

Comment:
Could you split this commit in two, please, first templatizing the text email and then adding the (also templated) HTML email variant separately?